ecocode-rules-specifications/src/main/script/prepare-resources.groovy

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env groovy
```

Comment:
Sorry I don't know at all groovy even if I read it, I understand.
If I recommend you to translate it into Python (easier for me), what do you think ?